lib/devise/migratable/model.rb

```diff
@@ -127,6 +127,11 @@ def feature_name
       module ClassMethods
         Devise::Models.config(self, :encryptor, :feature_class, :feature_name)
 
+        def feature_class_module
+          @feature_class_module ||=
+            feature_class.is_a?(String) ? const_get(feature_class) : feature_class
+        end
```

Comment:
I wonder if we didn't memoize this method if that would resolve the autoloading issue? Seems like it would to me since we would be re-constantizing it for each check. If we're really concerned about performance then I think we just need to add a `to_prepare` hook that re-memoizes this method.

Reply:
Added `reload_feature_class!` then in Rails `config/initializers/devise.rb` added `reloader.to_prepare` to reload.
```ruby
Rails.application.reloader.to_prepare do
  User.reload_feature_class!
end
```

Reply:
I think that's fine, it would be nice if the library could take care of the reloading but this will work.